spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample/pom.xml

```diff
@@ -15,7 +15,8 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-gcp-starter</artifactId>
+			<artifactId>spring-cloud-gcp-starter-vision</artifactId>
+			<version>1.1.0.BUILD-SNAPSHOT</version>
```

Comment:
Since it's our own starter, I think the version will be derived from the current project version.

Reply:
Gotcha. However I get this problem though when I try to run the sample after: 'dependencies.dependency.version' for org.springframework.cloud:spring-cloud-gcp-starter-vision:jar is missing.

Is there something missing in my pom.xml for the starter which prevents me from omitting the version?